### DISCUSSION TITLE
Limit-Einstellungen beim HM-600

### DISCUSSION THREAD

**Discussion by dl7ata** [tbnobody/OpenDTU#1347] [Q&A]
Moin,

softwaremäßig wird das Limit bis max. 100% akzeptiert (Funktion: setze Wechselrichterlimit). Was passiert, wenn man - mit Blick auf den 1.1.24 -  133% eingeben könnte? Gibt es die Hardware her, das Limit auf echte 800W zu erhöhen?
Lohnt es sich, den Parameter zu "pimpen"?

**Comment by Tiese** [comment]
Hi dl7ata!

Ähm, der HM-600 kann bestimmt nur max. 600W, sonst wäre es doch ein HM-800 #grübel#

**Reply by dl7ata** [reply]
Da die Baumaße identisch sind, könnte die Hardware - und evtl. auch die Firmware - ja auch für "mehr" ausgelegt sein.
Bei modernen Autos z.B. sind Zubehörfunktionen auch schon enthalten und werden nur noch freigeschaltet.

**Reply by Tiese** [reply]
Könnte sein - wie der Fernlicht-Assistent, beim Auto meiner Frau :-)
Wobei der GLA 220 auch nicht anders aussieht als der GLA 180 bzw. 200!

Dann müßtest du aber erst die Firmware flashen, damit er auch weiß, dass er ein HM800er sein soll.
